config/tap-driver.sh

```diff
@@ -402,7 +402,7 @@ function setup_result_obj(line)
   sub("^(not )?ok[ \t]*", "", line)
 
   # If the result has an explicit number, get it and strip it; otherwise,
-  # automatically assing the next progresive number to it.
```

Comment:
This file comes from an outside source and is already excluded in `.typos.toml` so should this change be dropped?